gcs/folder.go

```diff
@@ -27,11 +27,19 @@ func NewError(err error, format string, args ...interface{}) storage.Error {
 	return storage.NewError(err, "GCS", format, args...)
 }
 
-func NewFolder(bucket *gcs.BucketHandle, path string, contextTimeout int) *Folder {
-	return &Folder{bucket, path, contextTimeout}
+func NewFolder(bucket *gcs.BucketHandle, path string, contextTimeout int, normalizePrefix bool) *Folder {
+	return &Folder{bucket, path, contextTimeout, normalizePrefix}
 }
 
 func ConfigureFolder(prefix string, settings map[string]string) (storage.Folder, error) {
+	return configureFolder(prefix, settings, true);
```

Comment:
I think it would be better to add setting "GCS_NORMALIZE_PREFIX" and pass it with other settings to ConfigureFolder.

Reply:
Good idea, will implement it.